linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_creat/kubernetes/shared.yml

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_ocp,multi_platform_rhcos
+# reboot = true
+# strategy = disable
+# complexity = low
+# disruption = medium
+
+{{{ kubernetes_machineconfig_ospp_audit_rules() }}}
```

Comment:
Should we explicitly say in 
https://github.com/ComplianceAsCode/content/blob/2a632ada26e2365700032fcad3dca37ed208e637/linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_creat/rule.yml that `kubernetes` backend is off?

Reply:
we could... but as far as I can tell, this takes precedence.

Reply:
Yes, it does.
Was just concerned with possibility of content authors being confused.

Reply:
I leave it up to you.

Reply:
I'll leave it as-is.